gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
+
+#
+# Copyright © 2015-2021 the original authors.
+#
```

Comment:
I'm not sure why the Gradle files are included in the commit. Aren't they supposed to be included from our initial copies for the main repository, and therefore, not versioned?